src/Sulu/Bundle/SecurityBundle/Resources/config/services.xml

```diff
@@ -62,6 +62,7 @@
             <argument>%sulu_security.reset_password.mail.token_send_limit%</argument>
             <argument>%sulu_admin.email%</argument>
             <argument>%kernel.secret%%</argument>
```

Comment:
Btw that looks like a typo is it supposed to have two percent signs?

Reply:
Looks wrong to me. Think this should be only one `%`. Currently it may just added always a `%` to the secret.

Reply:
I've opened an issue for that. I wouldn't want to fix that here.